Formula/pgbouncer.rb

```diff
@@ -32,6 +32,13 @@ def install
     etc.install %w[etc/pgbouncer.ini etc/userlist.txt]
   end
 
+  def post_install
+    return if ENV["CI"]
```

Comment:
This should not be here

Reply:
There does need to be a post-install step that ensures that the directories for logging and PID are there, so I removed the part about CI. Let's see what the CI system has to say about that :)